pkg/deviceshifu/deviceshifumqtt/deviceshifumqttconfig.go

```diff
@@ -5,3 +5,9 @@ type ReturnBody struct {
 	MQTTMessage   string `json:"mqtt_message"`
 	MQTTTimestamp string `json:"mqtt_receive_timestamp"`
 }
+
+// RequestBody Body of mqtt's request by POST method
+type RequestBody struct {
+	MQTTTopic 	  string `json:"mqtt_topic"`
```

Comment:
We defined which mqttTopic to read from edge_device `CRD`, and for which mqttTopic to write to, I think we also need to define it in edge_device `CRD` for consistent.

Reply:
I have removed this line.